main.tf

```
@@ -13,8 +13,23 @@ resource "aws_config_configuration_recorder" "recorder" {
   count    = module.this.enabled ? 1 : 0
   name     = module.aws_config_label.id
   role_arn = local.create_iam_role ? module.iam_role[0].arn : var.iam_role_arn
+
   recording_group {
-    all_supported                 = true
+    all_supported                 = var.enable_exclusion == true || var.enable_inclusion == true ? false : true
```

I think this could be simplified to 
```suggestion
    all_supported                 = !var.enable_exclusion && !var.enable_inclusion
```